core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/metadata/OperatorMetadataGenerator.scala

```diff
@@ -92,6 +93,8 @@ object OperatorMetadataGenerator {
     jsonSchema.get("properties").asInstanceOf[ObjectNode].remove("operatorID")
     // remove operatorType from json schema
     jsonSchema.get("properties").asInstanceOf[ObjectNode].remove("operatorType")
+    // remove operatorVersion from json schema
+    jsonSchema.get("properties").asInstanceOf[ObjectNode].remove("operatorVersion")
```

Comment:
do we want to remove it? can you test if it exists in the json content

Reply:
yes, we do. If we don't, it would be one of the formly in the frontend.